pack/eoe/eoep.json

```diff
@@ -1033,221 +1670,766 @@
+        "skill_willpower": 1,
+        "slot": "Arcane",
+        "text": "Uses (9 charges).\n[action]: <b>Fight.</b> For this attck, you may use [willpower] instead of [combat], and you get +2 skill value. If you succeed, spend 1, 2, or 3 charges. Instead of its standard damage, this attack deals 1 damage for each charge spent. If you succeed by 0, lose 2 actions.",
+        "traits": "Spell.",
+        "type_code": "asset",
+        "xp": 4
+    },
+    {
+        "code": "08093",
+        "cost": 5,
+        "deck_limit": 2,
+        "faction_code": "guardian",
+        "faction2_code": "mystic",
+        "illustrator": "Robert Laskey",
+        "name": "Cyclopean Hammer",
+        "pack_code": "eoep",
+        "position": 93,
+        "quantity": 2,
+        "skill_combat": 2,
+        "skill_willpower": 2,
+        "slot": "Hand x2",
+        "text": "[action]: <b>Fight.</b> Add your [willpower] to your skill value for this attack. You deal +1 damage for this attack. If you succeed and the enemy is non-[[Elite]], you may move it one location away from you. (If you succeed by 3 or more, you instead deal +2 damage and may move the enemy up to two locations away from you.)",
+        "traits": "Item. Relic. Weapon. Melee.",
+        "type_code": "asset",
+        "xp": 5
+    },
+    {
+        "code": "08094",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "guardian",
+        "faction2_code": "survivor",
+        "flavor": "Twenty pounds of steel vengeance.",
+        "illustrator": "Tysen Johnson",
+        "name": "Sledgehammer",
+        "pack_code": "eoep",
+        "position": 94,
+        "quantity": 2,
+        "skill_combat": 1,
+        "slot": "Hand x2",
+        "text": "[action]: <b>Fight.</b> You get -1 [combat] and deal +1 damage for this attack.\n[action][action]: <b>Fight.</b> You get +2 [combat] and deal +2 damage for this attack.",
+        "traits": "Item. Tool. Weapon. Melee.",
+        "type_code": "asset",
+        "xp": 0
+    },
+    {
+        "code": "08095",
+        "cost": 4,
+        "deck_limit": 2,
+        "faction_code": "guardian",
+        "faction2_code": "survivor",
+        "flavor": "After what you've seen, you intend to sleep in it.",
+        "health": 3,
+        "illustrator": "Tony Foti",
+        "name": "Protective Gear",
+        "pack_code": "eoep",
+        "position": 95,
+        "quantity": 2,
+        "sanity": 3,
+        "skill_agility": 1,
+        "skill_willpower": 1,
+        "slot": "Body",
+        "text": "[reaction] When you draw a [[Hazard]] treachery, deal 1 damage and 1 horror to Protective Gear: Cancel that card's revelation effect.",
+        "traits": "Item. Armor.",
+        "type_code": "asset",
+        "xp": 2
+    },
+    {
+        "code": "08096",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "guardian",
+        "faction2_code": "survivor",
+        "flavor": "Twenty pounds of steel vengeance.",
```

Comment:
"Twenty-five" not "Twenty"

Reply:
That's pretty cute that they tweak the weight of the hammer for the leveled up flavor text. Totally missed that! XD